ValerieKC/wordle-redux
# Feature/theme switch

[Wordle](https://wordle-redux.web.app)

**_Description_**

- Add animation when keying in letter.
- Add functionality to switch between background theme.
- Add redux-persist to save theme state in localStorage.

**_Test Plan_**

- The background square will zoom in and zoom out while typing letter.
- Click switch theme button will toggle the web page theme between dark and light mode.
- Reload the webpage , the theme of the page should remain the same as the one set before reloading.
